examples/angular-cli/.gitignore

```
@@ -39,3 +39,6 @@ testem.log
 Thumbs.db
 addon-jest.testresults.json
 documentation.json
+/.yarn/*
+!/.yarn/releases
+!/.yarn/plugins
```

```suggestion
/.yarn/*
!/.yarn/releases
!/.yarn/plugins
!/.yarn/sdks
!/.yarn/versions
/.pnp.*
```

https://yarnpkg.com/getting-started/qa/#which-files-should-be-gitignored

---

And it was the wrong file 🙃  I did pay attention to when I cherry-picked that from the other branch.